### PR TITLE
New pop up component

### PR DESCRIPTION
**Pull request by ponochovny**

### CODE REVIEW

File: src/Components/NewPopUp/NewPopUp.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@import '../../Styles/index.scss';
+
+.NewPopUp {
+  &.modal {
+    .modal-header {
+      position: relative;
+      display: block;
+      // justify-content: space-between !important;
```
Comment (aslife) [inline src/Components/NewPopUp/NewPopUp.scss:8]: Remove unused code

File: src/Components/OpiumButton/Button.scss [hunk 2]
```diff
@@ -26,4 +33,322 @@
...
+                    &:disabled {
+                        background-color: transparent;
+                        color: rgba(white, .5);
+                        border-color: rgba(white, .5);
+                    }
+                }
+                &-LIGHT {
+                    background-color: #0a0a1e;
+                    color: white;
+                    border-color: #0a0a1e;
```
Comment (aslife) [inline src/Components/OpiumButton/Button.scss:79]: Use variables everywhere

File: src/Components/OpiumButton/Button.scss [hunk 3]
```diff
@@ -26,4 +33,322 @@
...
+                        background-color: rgba(10, 10, 30, 0.5);
+                        color: white;
+                        border-color: transparent;
+                    }
+                    &:disabled {
+                        background-color: rgba(#0A0A1E,.5);
+                        color: rgba(#0A0A1E,.5);
+                        border-color: transparent;
+                    }
+                    &:disabled {
```
Comment (aslife) [inline src/Components/OpiumButton/Button.scss:141]: Duplicate code

File: src/Components/OpiumButton/Button.scss [hunk 4]
```diff
@@ -26,4 +33,322 @@
...
+                }
+            }
+            &.green {
+                &.color-scheme {
+                    &-DARK {
+                        border-color: #2ECD94;
+                        color: #2ECD94;
+                        &:hover {
+                            background-color: #2ECD94;
+                            color: #0A0A1E;
```
Comment (aslife) [inline src/Components/OpiumButton/Button.scss:196]: Use variables

File: src/Components/OpiumButton/index.tsx [hunk 5]
```diff
@@ -95,21 +104,33 @@ const OpiumButton: FC<Props> = (props: Props) => {
...
-    monoStyles.backgroundColor = backgroundColor[variant as TVariant].disabled
-    monoStyles.borderColor = borderColor[variant as TVariant].disabled
-    monoStyles.color = color[variant as TVariant].disabled
-    rainbowLabel.color = color['rainbow'].disabled
-  }
+  // if (hover) {
+  //   if (
+  //     !(variant === 'secondaryRed' ||
+  //     variant === 'secondaryGreen' ||
+  //     variant === 'secondaryBlue')
```
Comment (aslife) [inline src/Components/OpiumButton/index.tsx:111]: Remove unused code

**Review by aslife — COMMENTED** [review]
Close button is white (light theme). It is not correct
Popup should has a white background

File: src/Components/OpiumButton/Button.scss (hunk 3 above)
Comment (ponochovny) [reply src/Components/OpiumButton/Button.scss:140]: It Will be fixed soon, now it's hard to predict the structure of a mixin

**Pull request merged**